lib/email.rb

```diff
@@ -0,0 +1,26 @@
+class Email
+  def initialize(user)
+    @address = user.email
+    @name = user.email.split(/@/).first
```

Comment:
Is there an easier way to get the users name? Could it be stored it in the session object at the controller, and sent through?